it/src/it/fabric8-jenkins-github/expected/k8s-template.yml

```diff
@@ -1082,7 +1471,7 @@ objects:
           fabric8.io/git-branch: release-v2.2.335
           fabric8.io/scm-devcon-url: scm:git:git@github.com:fabric8io/fabric8-devops.git/configmapcontroller
           fabric8.io/scm-tag: fabric8-devops-2.0.1
-          maven.fabric8.io/source-url: jar:file:/home/rohan/.m2/repository/io/fabric8/devops/apps/configmapcontroller/2.2.335/configmapcontroller-2.2.335.jar!/META-INF/fabric8/kubernetes.yml
+          maven.fabric8.io/source-url: jar:file:/home/hshinde/.m2/repository/io/fabric8/devops/apps/configmapcontroller/2.2.335/configmapcontroller-2.2.335.jar!/META-INF/fabric8/kubernetes.yml
```

Comment:
Hmm. This is creating problems .i wonder if we can just remove such kind of portions from these files which are related to local path of files.